ouroboros-network/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 * Disable light peer sharing if peer sharing is not enabled.
 * Do not use light peer sharing in `node-to-client` case (which leads to a memory leak).
+* Less aggresive churning of established and known peers.
```

Comment:
I think you need to rebase the branch since the other two things were released in `0.9.1`; It's also possible that I forgot to update the `CHANGELOG`, if that's the case could you fix it?